File: App.Models/DTOs/UpdateRquests/FunctionUpdateRequest.cs
using MaxV.Common.Model.DTOs;

namespace App.Models.DTOs.UpdateRquests
{
    public class FunctionUpdateRequest : BaseUpdateRequest<string>
    {
        public string Name { get; set; }
        public string Url { get; set; }
        public int SortOrder { get; set; }
        public string ParentId { get; set; }
        public string Icon { get; set; }
    }
}

File: admin-angular/src/app/models/update-requests/function-update-request.ts
import { AutoMap } from "@automapper/classes";
import { BaseUpdateRequest } from "..";

export class FunctionUpdateRequest extends BaseUpdateRequest {
    @AutoMap()
    name: string;
    @AutoMap()
    url: string;
    @AutoMap()
    sortOrder: number;
    @AutoMap()
    parentId: string;
    @AutoMap()
    icon: string;
}
